package.json

```diff
@@ -15,7 +15,7 @@
     "codecov": "^3.1.0",
     "mocha": "^5.2.0",
     "nyc": "^13.1.0",
-    "request": "^2.46.0"
+    "request": "2.46.0"
```

Comment:
According to Snyk this version contains a vulnerability. Let's keep the ^.
```suggestion
    "request": "^2.46.0"
```

Reply:
Thank you so much!